phpmd.xml

```diff
@@ -22,6 +27,11 @@
     <rule ref="rulesets/naming.xml">
         <exclude name="ShortVariable" />
     </rule>
+    <rule ref="rulesets/naming.xml/ShortVariable">
+        <properties>
```

Comment:
I tried other styles but this seems to be the only one that works. First import a rule set, excluding one rule. Then import the excluded rule with specific options applied.